src/Components/Login.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import { signInUser } from '../state/actions/reduxTokenAuthConfig'
+import { connect } from 'react-redux'
+import LoginForm from './LoginForm'
+
+class Login extends Component {
+  state = {
+    email: '',
+    password: '',
+    errorMessage: ''
+  }
+
+  inputChangeHandler = (e) => {
```

Comment:
Please be consistent with your code formatting, make sure everyone uses prettier.